doc/source/user_guide/indexing.rst

```diff
@@ -237,7 +237,7 @@ new column. In 0.21.0 and later, this will raise a ``UserWarning``:
 
     In [1]: df = pd.DataFrame({'one': [1., 2., 3.]})
     In [2]: df.two = [4, 5, 6]
-    UserWarning: Pandas doesn't allow Series to be assigned into nonexistent columns - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute_access
```

Comment:
Can you revert this one as well? (it's in the output of code, which should still return the absolute url)